components/Contest.tsx

```diff
@@ -3,27 +3,35 @@ import { IFilter } from './helpers/interfaces'
 interface Contest {
   name: string
   description: string
-  date_end: string
+  dateEnd: string
   status: IFilter
 }
```

Comment:
you don't need this, you already have the `ContestType` interface on the helpers, it's the same interface